openmc/source.py

```diff
@@ -198,6 +198,43 @@ def to_xml_element(self):
             element.append(self.time.to_xml_element('time'))
         return element
 
+    def plot_energy(self, n_samples=10000, seed=None, axes=None):
```

Comment:
I would expose the `hist` arguments for more flexibility
```suggestion
    def plot_energy(self, n_samples=10000, seed=None, axes=None, **kwargs):
```